### PR TITLE
Ui/move windows 7

### PR DESCRIPTION
**Pull request by marijnvdwerf**

### CODE REVIEW

**Comment by AaronVanGeffen** [comment]
There'll be no more windows left in libopenrct2 when this is merged, right? Impressive work!

**Comment by marijnvdwerf** [comment]
Yes, after this is merged no actual windows remain in libopenrct2. The code behind the windows still is though.

File: src/openrct2/input.cpp [hunk 1]
```diff
@@ -0,0 +1,99 @@
+#pragma region Copyright (c) 2014-2017 OpenRCT2 Developers
```
Comment (Gymnasiast) [inline src/openrct2/input.cpp]: Should be capitalised: `Input.cpp`.

File: src/openrct2/input.cpp [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+ *
+ * A full copy of the GNU General Public License can be found in licence.txt
+ *****************************************************************************/
+#pragma endregion
+
+#include "audio/audio.h"
+#include "Context.h"
+#include "input.h"
+
+typedef struct rct_mouse_data {
```
Comment (Gymnasiast) [inline src/openrct2/input.cpp]: This brace should be on a new line.

**Comment by Gymnasiast** [comment]
Overall, this looks pretty decent. Found two minor niggles and it will obviously have to be rebased.

File: src/openrct2-ui/input/MouseInput.cpp [hunk 3]
```diff
@@ -0,0 +1,1551 @@
...
+#include <openrct2/world/banner.h>
+#include <openrct2/world/map.h>
+#include <openrct2/world/sprite.h>
+#include <openrct2/world/scenery.h>
+#include <openrct2/OpenRCT2.h>
+#include <openrct2/core/Math.hpp>
+#include <openrct2/core/Util.hpp>
+#include <openrct2-ui/windows/Window.h>
+
+typedef struct rct_mouse_data {
```
Comment (Gymnasiast) [inline src/openrct2-ui/input/MouseInput.cpp]: Please reformat this file.

Comment (AaronVanGeffen) [reply src/openrct2-ui/input/MouseInput.cpp]: I recall someone using the excuse "This deliberately skips reformatting, in order to get stuff to C++ quicker." elsewhere… :wink:

Comment (Gymnasiast) [reply src/openrct2-ui/input/MouseInput.cpp]: Those are not refactor PRs. This one is.

Comment (marijnvdwerf) [reply src/openrct2-ui/input/MouseInput.cpp]: This isn't a refactor PR. It just moves stuff.

Comment (Gymnasiast) [reply src/openrct2-ui/input/MouseInput.cpp]: How is that not refactoring? You're moving it to split UI and engine bits.
Anyway, I'm not going obstruct the PR for it, but I'd be disappointed if you didn't do at least an autoformat to correct braces and spaces.

**Pull request merged**